### ISSUE TITLE
UI/UX Stretch Goals - Part 2

### ISSUE THREAD

**Issue by kaloster** [chanzuckerberg/single-cell-data-portal#7183]
This is part 2 of the UI/UX updates for side by side.

Add Cell Type info to new info panel created [here](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-explorer/947)

- Add cell type endpoint
- Add tab to info panel
- Add info icon to cell types to trigger cell type tab
- Add search box to cell tab
- https://github.com/chanzuckerberg/single-cell-explorer/assets/8716829/f242406e-4ac3-41e8-8482-4fb9b96f0278

figma: https://www.figma.com/design/eJ3acPxG04PKVlom6eS8ft/Spatial?node-id=0-1&t=yAWU6j6wSLKV9eIJ-0

**Comment by tihuan** [comment]
@kaloster what's the status of this ticket again? Thank you!

**Comment by tihuan** [comment]
@kaloster realized that this ticker is blocked by not having easy access to cell ontology id ([ticket](https://github.com/chanzuckerberg/cellxgene-ontology-guide/issues/206)). Moved to post launch epic
